plugins/README.md

```diff
@@ -2,10 +2,39 @@
 
 For step-by-step instructions on using and making plugins with Builder, see the official documentation, starting with [Intro to Plugins](https://www.builder.io/c/docs/plugins-intro).
 
-Example plugins:
+## Example plugins:
 
 - [Rich text](rich-text)
 - [Cloudinary](cloudinary)
 - [Shopify](shopify)
 - [Async dropdown](async-dropdown)
 - [Campaign app](example-app-campaign-builder)
+
+
+## Plugin Testing/Deployment Steps
+
+Our plugins are hosted on npm under package @builder.io . You can check this url to see details of any plugin 
+https://www.npmjs.com/package/@builder.io/[plugin-name]
+
+- Test your changes by running the plugin locally and raise a PR
```

Comment:
It would be great if we mention how they can 'run' the plugin locally and have it installed as local plugin via plugin settings in the webapp